rom/dev/src/flow/cold_reset/fw_processor.rs

```diff
@@ -65,8 +65,8 @@ impl FirmwareProcessor {
             // sha256
             sha256: &mut env.sha256,
 
-            // SHA2-384 Engine
-            sha384: &mut env.sha384,
+            // SHA2-512/384 Engine
+            sha2_512_384: &mut env.sha2_512_384,
```

Comment:
minor nit: I would probably just call this driver sha2_512 and then also include sha384 functions on it. Since SHA384 effectively is just a variant of sha512 (different IV + truncated). But not a huge deal.

Reply:
I named as such it to keep the name consistent with sha sha2_512_384_acc.